### PR TITLE
Fixed dismiss and back route in whole pages, fixed route type err in …

### PR DESCRIPTION
**Pull request by erenkan**
…auth,

### CODE REVIEW

File: scripts/routes/auth.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { NativeStackRouter as StackRouter, Route } from '@smartface/router';
+import { NativeStackRouter, NativeStackRouter as StackRouter, Route } from '@smartface/router';
```
Comment (alimoncul) [inline scripts/routes/auth.ts]: BaseRouter should be imported right, you can use the imports from page I think

File: scripts/pages/pgWelcome.ts [hunk 2]
```diff
@@ -16,7 +19,10 @@ export default class PgWelcome extends withDismissAndBackButton(PgWelcomeDesign)
     }
     onShow() {
         super.onShow();
-        this.initDismissButton(this.router);
+        this.initDismissButton(this.router, {
+            color: themeService.getStyle('.sf-headerBar.itemColor'),
+            image: Image.createFromFile(`images://${image}`)
```
Comment (alimoncul) [inline scripts/pages/pgWelcome.ts]: Does all dismiss icons changed? If so we should change default one

**Review by alimoncul — CHANGES_REQUESTED** [review]

File: scripts/routes/auth.ts (hunk 1 above)
Comment (erenkan) [reply scripts/routes/auth.ts]: OK.

File: scripts/pages/pgWelcome.ts (hunk 2 above)
Comment (erenkan) [reply scripts/pages/pgWelcome.ts]: OK.

**Pull request merged**